analytics/upress_reporting/models/reports/total_usage.py

```diff
@@ -1,6 +1,41 @@
+import csv
+
 from models.reports.counter_5_report import Counter5Report
 
 
 class TotalUsageReport(Counter5Report):
-    def __init__(self, publisher):
-        super().__init__(publisher)
+    def __init__(self, *args):
+        super().__init__(*args)
+    
+    def build_report(self, events):
+        print("Building total usage report...")
+
+        header = self.build_header()
+
+        if len(events) > 0:
+            columns, final_data = self.aggregate_interaction_events(events)
+            csv_file_name = f"{self.publisher}_total_usage_{self.created}.csv"
+            
+            with open(csv_file_name, 'w') as csv_file:
+                writer = csv.writer(csv_file, delimiter="|")
+                for key, value in header.items():
+                    writer.writerow([key, value])
+                writer.writerow([])
+                writer.writerow(columns)
+                for title in final_data:
+                    writer.writerow(title.values())
+
+            print("Total usage report generation complete!")
+        else:
+            print("No events found in reporting period!")
```

Comment:
Why are print statements used here instead of logging statements? This question applies to your previous PR as well, but I didn't notice it until now.

Reply:
Since this code is run locally, we just want to print to the console!

Reply:
Ok that makes sense